packages/cli/src/telemetry/index.ts

```diff
@@ -0,0 +1,126 @@
+/// <reference lib="dom" />
```

Comment:
??

Reply:
This is necessary for the Firebase types. It uses some types from the browser. I have added a comment.